aibolit/scripts/02-calculate-metrics.py

```diff
@@ -0,0 +1,27 @@
+import os
+import sys
+
+if not os.path.isdir('02'):
+    os.makedirs('02')
+f = open('02/metrics.csv', 'w+')
```

Comment:
@zuoqin let's give this a more descriptive name like `file_metrics`. Also, let's use Python `with` statement block so we don't have to close things manually:
https://www.pythonforbeginners.com/files/with-statement-in-python